src/test/resources/karate-config.js

```diff
@@ -1,12 +1,25 @@
 function config() {
+
     var env = karate.env; // get system property 'karate.env'
     karate.log('karate.env system property was:', env);
     karate.configure('ssl',true);
+
+    DB_URL: DB_URL,
+    DB_USERNAME:DB_USERNAME,
+    DB_PASSWORD:DB_PASSWORD,
+    DB_DRIVER:DB_DRIVER,
+    URL_PATH:URL_PATH
+
     if (!env) {
       env = 'dev';
     }
-    var data = read('classpath:karate/support/env_data.json');
+    var data = read('classpath:support/env_data.json');
     var env_data = data[env];
+    var  DB_URL = data.DB_URL;
+    var  DB_USERNAME = data.DB_USERNAME;
+    var  DB_PASSWORD = data.DB_PASSWORD;
+    var  DB_DRIVER = data.DB_DRIVER;
+    var  URL_PATH = data.URL_PATH;
```

Comment:
this works? we shouldn't be assigning db values at all data here

anyway, I think we shouldn't be doing this to load env vars on the project and should use instead some function to automatically load all the envs on some other file to avoid editing this file every time a new one is created

check something like this
```
var data = karate.read('classpath:support/env_data.json');
config = karate.toMap(data)
```

Reply:
this file was ignored and the .json was removed

Reply:
not sure what you did here, why adding the karate-config on gitignore? thats a needed file and should be on the repo (and although you ignored it its still here)

Reply:
The karate-config.js file cannot be ignored, the first thing maven does is read this file to run all the tests and the KarateRunner

Reply:
Ok fixed